### ISSUE TITLE
XCode 14.3 Clang command failing

### ISSUE THREAD

**Issue by mohammedDehairy** [open-obfuscator/o-mvll#10]
Followed all instructions :

```
Using O-MVLL with Xcode is a bit easier than Android since we don’t need to deal with different libstdc++/libc++. To enable O-MVLL, one needs to go to:

Build Settings > Apple Clang - Custom Compiler Flags > Other C/C++ Flags

and add -fpass-plugin=<path>/omvll_xcode_14.dylib. We also need to disable the legacy pass manager with: -fno-legacy-pass-manager.

Finally, we can create an omvll.yml file next to the *.xcodeproj file which defines OMVLL_PYTHONPATH and OMVLL_CONFIG.

Et voila :)
```
but when i try to build using xcode i get this error:

![Screenshot 2023-05-08 at 08 56 59](https://user-images.githubusercontent.com/2619933/236755702-dab75a0a-9c6b-45f3-b492-66e6d06f0ddd.png)

it seems clang crashed

```
Crashed Thread:        0  Dispatch queue: com.apple.main-thread

Exception Type:        EXC_BAD_ACCESS (SIGKILL (Code Signature Invalid))
Exception Codes:       UNKNOWN_0x32 at 0x0000000118000000
Exception Codes:       0x0000000000000032, 0x0000000118000000
```

Can you help me?

Target: iOS
O-MVLL Version: OMVLL Version: 1.1.0-86087c7 / 14.0.0git (d5f117e38620783fc095e5620416bdae08554588)
Compilation of O-MVLL: on my own

**Comment by romainthomas** [comment]
This is likely because of M1 code signature enforcement. I know that it is not ideal but I had to disable SIP.

**Comment by mohammedDehairy** [comment]
Thanks @romainthomas for the quick response!

Yeah, I don’t think I can disabled SIP, any other possible work around?

if that’s the only way, I am afraid we’ll have to use some other obfuscation tool

**Comment by JordiVentayol** [comment]
Hi @mohammedDehairy , I was able to solve the error by signing the library: `codesign --force --deep -s <identity> omvll_xcode_14.dylib` .

Now I'm running into another problem:

<img width="258" alt="jvmxcode" src="https://github.com/open-obfuscator/o-mvll/assets/53511252/c7c98b85-3317-44c8-85d7-59a2f6d02bd5">

 but you can try if the same thing happens to you.

**Comment by JordiVentayol** [comment]
> Hi @mohammedDehairy , I was able to solve the error by signing the library: `codesign --force --deep -s <identity> omvll_xcode_14.dylib` .
> 
> Now I'm running into another problem:
> 
> <img alt="jvmxcode" width="258" src="https://user-images.githubusercontent.com/53511252/271923991-c7c98b85-3317-44c8-85d7-59a2f6d02bd5.png">
> but you can try if the same thing happens to you.

Okay, this error was due to an IndentationError in my ovmll_config.py, so what was suggested above should solve the error.

**Comment by marcobrador** [comment]
Code signature was implemented as part of [v1.0.3](https://github.com/open-obfuscator/o-mvll/releases/tag/1.0.3)
